doc/guides/backporting-to-release-lines.md

```diff
@@ -68,25 +66,24 @@ hint: with 'git add <paths>' or 'git rm <paths>'
 hint: and commit the result with 'git commit'
 ```
 
-* Make the required changes to remove the conflicts, add the files to the index
-  using `git add`, and then commit the changes. That can be done with
-  `git cherry-pick --continue`.
-* Leave the commit message as is. If you think it should be modified, comment
-  in the Pull Request.
-* Make sure `make -j4 test` passes
-* Push the changes to your fork and open a pull request.
-* Be sure to target the `v7.x-staging` branch in the pull request.
-
-### Helpful Hints
-
-* Please include the backport target in the pull request title in the following
-  format: `(v7.x backport) <commit title>`
-  * Ex. `(v4.x backport) process: improve performance of nextTick`
-* Please check the checkbox labelled "Allow edits from maintainers".
-  This is the easiest way to to avoid constant rebases.
-
-In the event the backport pull request is different than the original,
-the backport pull request should be reviewed the same way a new pull request
-is reviewed.
+5. Make the required changes to remove the conflicts, add the files to the index
+   using `git add`, and then commit the changes. That can be done with
+   `git cherry-pick --continue`.
+6. Leave the commit message as is. If you think it should be modified, comment
+   in the Pull Request.
+7. Make sure `make -j4 test` passes.
+8. Push the changes to your fork
+9. Open a pull request:
+   1. Be sure to target the `v7.x-staging` branch in the pull request.
+   2. Include the backport target in the pull request title in the following
+      format — `(v7.x backport) #<PR_NUMBER> - <commit title>`.
+      Example: `(v7.x backport) #10157 - process: improve performance of nextTick`
+   3. Check the checkbox labelled "Allow edits from maintainers".
+   4. In the description add a referance to the original PR
+10. Replace the `backport-requested-v7.x` label on the original PR
+    with `backported-to-v7.x`.
+
+*Note*: If the backport pull request is different than the original, it should
+be reviewed the same way a new pull request is reviewed.
```

Comment:
I have no Idea, just reformatted from previous text.
Maybe this alludes to the case when the cherry-pick had conflicts that had to be manually resolved.
Or cases like https://github.com/nodejs/node/pull/13750 where I had to add unrelated code i.e. https://github.com/nodejs/node/pull/13750/commits/d429bc699e4737056656696ab47cdaf58f5b1ca8